### ISSUE TITLE
Controls not visible on embedded mode

### ISSUE THREAD

**Issue by amoncaldas** [GIScience/ors-map-client#167]
Some controls (layer selector, zoom, accessibility etc) are not visible in embedded mode
https://openrouteservice.org/new-embedded-mode-for-openrouteservice-map-client/
